database/types.go

```diff
@@ -16,3 +16,17 @@ type Get struct {
 type Set struct {
 	DB *Database
 }
+
+type ColumnInfo struct {
+	Name string
+	Type string
+}
+
+type TableColumn struct {
```

Comment:
Can you add comments here? Also ColumnInfo seems redundant here. We already have name and type in TableColumn. Or is it for something else? You can use comments here to specify what each struct will be used for.

Reply:
You`re right! Don`t know why i have both. Refactored it to only use TableColumn.